### ISSUE TITLE
Provide method(s) for version bump 

### ISSUE THREAD

**Issue by dsentker** [ergebnis/version#7]
I really like the idea and the package. How about provide a method for increasing the version?

`$version->patch->increase()`

I can provide a PR if this sounds useful.

**Comment by localheinz** [comment]
Sounds good, @dsentker!

If we want to bump versions, then perhaps we need something like this:

```php
<?php

declare(strict_types=1);

use Ergebnis\Version;

$version = Version\Version::fromString('1.2.3');

$one = $version->bumpMajor();

echo $one->toString(); // 2.0.0

$two = $version->bumpMinor();

echo $two->toString(); // 1.3.0

$three = $version->bumpPatch();

echo $three->toString(); // 1.2.4
```

What do you think?

**Comment by dsentker** [comment]
You're right, your proposal is easier to read. I'll add a PR when Christmas days are over 🙂

**Comment by localheinz** [comment]
No worries, @dsentker - enjoy the holidays!

**Comment by localheinz** [comment]
Implemented this in #13, @dsentker!

**Comment by dsentker** [comment]
@localheinz You were significantly faster than me. Thank you. My solution wouldn't have been as elegant as yours either.
